content/documentation/overview/index.adoc

```diff
@@ -28,7 +28,7 @@ A Microservice Architecture breaks up the monolith into many smaller pieces that
 A service mesh can now provide these services on a platform level and frees the application writers from those tasks. Routing decisions are done at the mesh level.
 
 
-icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali works with Istio to visualise the service mesh topology, features like circuit breakers or request rates.
+icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali works with Istio, in OpenShift or Kubernetes, to visualize the service mesh topology, features like circuit breakers or request rates.
```

Comment:
I wonder if Openshift should be replaced with OKD @JStickler ? (but it should not block this PR, we can change it later)
@jotak how about
"Kiali works with Istio, in OpenShift or Kubernetes, to visualize the service mesh topology, provide visibility into features like circuit breakers, request rates and more."

Reply:
Ok - and I would add after that: "It offers insights in mesh components at different levels, from abstract Applications to Services and Workloads."

Also I would remove the "lightbulb" since we're moving from a small hint to a broader explanation of what is kiali.

Ok?

Reply:
@jotak good :)
I would say "It offers insights about/to the mesh components", I don't think that "in" is the right one here.